.vscode/launch.json.template

```diff
@@ -35,5 +35,35 @@
             "mode": "test",
             "program": "${fileDirname}"
         },
+
+// Run process agent checks manually
+// If necessary, add arguments "-config", "<custom datadog.yaml file path>" to override the default process agent configuration
+        
+        {
+            "name": "process",
+            "type": "go",
+            "request": "launch",
+            "mode": "exec",
+            "program": "${workspaceRoot}/bin/process-agent/process-agent",
+            "env": {},
+            "showLog": true, 
+
+            "args": [
+                "check",
+                "process"
+            ]
+        },
+        {
+            "name": "processDiscovery",
+            "type": "go",
+            "request": "launch",
+            "mode": "exec",
+            "program": "${workspaceRoot}/bin/process-agent/process-agent",
+            "env": {},
+            "args": [
+                "check",
+                "process_discovery"
+            ]
+        }
```

Comment:
I'm not too familiar with vscode. I think we should keep this out of the change set as it's very IDE specific. If we want this to go in we should isolate it into its own PR and provide documentation on how its used.